app/containers/AppWrapper/Box.js

```diff
@@ -2,30 +2,20 @@ import styled from 'styled-components';
 
 import { singleCommunityColors } from 'utils/communityManagement';
 
-import {
-  HEADER_HEIGHT,
-  LOADER_HEIGHT,
-  MOBILE_HEADER_HEIGHT,
-} from 'containers/Header/constants';
+import { HEADER_HEIGHT, LOADER_HEIGHT, MOBILE_HEADER_HEIGHT } from 'containers/Header/constants';
 
 import { LEFT_MENU_WIDTH } from './constants';
 
 const colors = singleCommunityColors();
 
 const Main = styled.div`
-  background: ${colors.mainBackground
-    ? colors.mainBackground
-    : 'rgb(234, 236, 244)'};
+  background: ${colors.mainBackground ? colors.mainBackground : 'rgb(234, 236, 244)'};
```

Comment:
why not use: `background: ${colors.mainBackground || 'rgb(234, 236, 244)'}; `?

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1402/commits/78897b69b4ad523ef680321471acfef0cbeb7f4f